scripts/run-build.ps1

```diff
@@ -44,7 +46,7 @@ $env:CHANNEL=$env:RELEASE_SUFFIX
 # Use a repo-local install directory (but not the artifacts directory because that gets cleaned a lot
 if (!$env:DOTNET_INSTALL_DIR)
 {
-    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows"
+    $env:DOTNET_INSTALL_DIR="$PSScriptRoot\..\.dotnet_stage0\Windows\$Architecture"
```

Comment:
Does this need to be updated here?

https://github.com/dotnet/cli/blob/rel/1.0.0/scripts/dotnet-cli-build/Utils/DotNetCli.cs#L38

Also, should we fork between Windows and Non-Windows like this? Would it make sense to add the architecture to the install path for NonWindows?